.circleci/config.yml

```
@@ -2,7 +2,7 @@
 # syntax, such as anchors, will be fixed automatically.
 version: 2.1
 orbs:
-  shared: getoutreach/shared@2.21.0
+  shared: getoutreach/shared@dev:first
```

suggestion: we may want to make this a stable version instead of a expiring dev version.
```suggestion
  shared: getoutreach/shared@2.24.0
```